.github/.jira_sync_config.yaml

```diff
@@ -10,14 +10,11 @@ settings:
   # (Optional) Jira project components that should be attached to the created issue
   # Component names are case-sensitive
   components:
-    - IoT
-    - DACH TT
+    - K8s charm
```

Comment:
Please ensure that this component is added in Jira (case sensitive)

You can ping me in MM

Reply:
done